packages/web-pkg/src/composables/clientService/useClientService.ts

```diff
@@ -0,0 +1,6 @@
+import { getCurrentInstance } from '@vue/composition-api'
+import { ClientService } from '../../services'
+
+export const useClientService = (): ClientService => {
+  return (getCurrentInstance().proxy as any).$clientService
```

Comment:
The clientService is available as a singleton instance which can be directly imported. Could you explain why you prefer this solution over the direct import? I know that singleton is kind of an antipattern, but making it available via the vm is not really better. 😅 (proper dependency injection would be nice, but not that much of a priority right now)

Reply:
To me it's utterly confusing to use these singletons in tests, one has to know way too much about how these tests are executed, when contexts are reset etc. Overriding per vm instance is way clearer to me.

Yes, I agree standard injecting everything in all components is an anti-pattern as well, but still less bad IMHO. Can be replaced with explicit injection when time permits more easily... (this way instances are only injected into components and need to be passed into every other kind of "module" explicitly, as you can see in the avatarUrl helper. The changes in the corresponding test also show why I think this is cleaner: no messing with the global Vue instance anymore)